src/content/blog/2024-08-22-the-ugrc-api-address-location-lane.mdx

```diff
@@ -0,0 +1,45 @@
+
+<Image
+  src={centerlineImage}
+  width={800}
+  alt="A line indicating fiction street with a circle at the 25% location along the segment"
+/>
+
+To understand why, lets break down what happens with a match on the road centerlines with an example. Consider the input street address is `125 South Fiction Street`. The road centerline data has a segment representing Fiction Street with attributes defining the address range for that segment e.g. 100-200. This guarantees that any address on this segment will have numbers between 100 and 200. Since 125 falls within the address range of the segment, the road centerline address locator will provide a match that is 25% along the line segment in the positive direction. However, the exact location of the address along the segment is not known; It may not exist in terms of a structure, mail delivery, or the availability of 911 services. **If you are solving or validating for mail delivery, 911 response, voter registration, etc., this is not an accurate solution.** You may need to evaluate services provided by the [USPS](https://www.usps.com/) or [Smarty](https://smarty.com/) to validate mail delivery.
+
+Address location matches solved by the address points do represent a geographic location that has been assigned a US Postal Service (USPS) address by the local address authority (i.e., county or municipality) but does not necessarily receive mail. For example, mail may be delivered to a post office box.
+
+## The UGRC benefit
+
+The UGRC API address locating endpoint is a free service that provides the most comprehensive government-sourced GIS address data available for Utah. We are able to communicate with our address authorities to fix any issues you may encounter with the data quickly. This feedback loop ensures that the data is always improving and that you are getting the best address locating service possible. We provide excellent customer service and timely responses to your questions and concerns. We are here to help you get the most out of our data.
+
+Please carefully evaluate if the UGRC API is the right choice for your addressing needs. If address verification and validation are terms that come to mind, distill what that means to you and your needs. If you are looking for a generalized location for analysis, the UGRC API is a great choice. If you are looking for a precise location for mail delivery, 911 response, voter registration, etc., you may need to evaluate other services.
```

Comment:
```suggestion
Please carefully evaluate if the UGRC API is the right choice for your addressing needs. If you are looking for a generalized location for analysis, the UGRC API is a great choice. If you are looking for a precise location for mail delivery you may need to evaluate other services.
```

Reply:
i struggle with saying that our data is not suitable for 911 and elections, b/c it is used, in a sense, in both of those systems. 

from my understanding, our address data supplements the voter registration data and is the main input for 911 call routing. 

maybe it's good enough to just say that we know our data is not perfect, but are working with the local addressing authorities to improve it and that 911 and elections are the main drivers that help us improve the data.

Reply:
Right I hear that. But that "in a sense" caveat should be explained and is the intention of this blog post. Can you elaborate on that and update your suggestion? Maybe we add 2 more headings for voting and 911 to explain how they are used?

Reply:
Sure, I like that. The main takeaway should be that our data is used in conjunction with other data sources in these critical business systems. 

For 911:
we could mention that 911 uses our address data (address points and road centerlines) to route 911 calls originating from a landline (with an attached home address). We work with the service providers to ensure that home addresses for their landline customers are in our system.  
Additionally, in dispatch centers, our address data is used to log incident reports and to route emergency response, however, some dispatch centers have supplemental address data that is used in conjunction with our data. Their supplemental data often has a local nuance, to account for their local vernacular. 

For elections:
The UGRC address locator is an input into the voter registration database. Our data is used to locate voters. However, our data is supplemented with Smarty as well as local knowledge/data: clerks use UGRC address data as a starting point, but can modify the x,y locations in their local copy as needed).  Smarty also provides information regarding residential or commercial)

Reply:
note: i modified my original suggestion.

Reply:
i took what you added and reorganized. I'm going to resolve this and set the new draft as ready for review again.